library/Icinga/Web/RememberMeUserDevicesList.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/* Icinga Web 2 | (c) 2020 Icinga GmbH | GPLv2+ */
+namespace Icinga\Web;
+
+use ipl\Html\BaseHtmlElement;
+use ipl\Html\Html;
+use ipl\Web\Url;
+use ipl\Web\Widget\Icon;
+use ipl\Web\Widget\Link;
+
+class RememberMeUserDevicesList extends BaseHtmlElement
+{
+    protected $tag = 'table';
+
+    protected $defaultAttributes = [
+        'class'             => 'common-table',
+        'data-base-target'  => '_self',
+        'title'             => 'click to remove this cookie from the database'
```

Comment:
Please remove the title here. It's the link of each row that's responsible to remove a session, not the entire table. (which I can't click on anyway)